jwql/utils/monitor_template.py

```diff
@@ -105,7 +105,7 @@ def monitor_template_main():
     #      'suffix': 'uncal'}
 
     # Example of locating a dataset in the filesystem
-    filesystem = get_config()['filesystem']
+    filesystem = get_config()['public_filesystem']
```

Comment:
Might be worth mentioning the proprietary filesystem here as well just so developers don't forget to search it?